### PR TITLE
Test on multiple terraform versions

### LINKED ISSUE #35: update terraform version and code to 1.1.4 (latest)

### PR DESCRIPTION
**Pull request by michalbagrowski**
resolve https://github.com/aeternity/terraform-aws-api-gateway/issues/35

### CODE REVIEW

File: test/test.tf [hunk 1]
```diff
@@ -33,7 +33,7 @@ module "test_nodes_sydney" {
 }
 
 module "test_nodes_sydney_channels" {
-  source            = "github.com/aeternity/terraform-aws-aenode-deploy?ref=master"
+  source            = "github.com/aeternity/terraform-aws-aenode-deploy?ref=test-on-multiple-terraform-versions"
```
Comment (michalbagrowski) [inline test/test.tf]: will be fixed after https://github.com/aeternity/terraform-aws-aenode-deploy/pull/31 merge as master have issues

File: test/test.tf [hunk 2]
```diff
@@ -7,7 +7,7 @@ locals {
 }
 
 module "test_nodes_sydney" {
-  source            = "github.com/aeternity/terraform-aws-aenode-deploy?ref=master"
+  source            = "github.com/aeternity/terraform-aws-aenode-deploy?ref=test-on-multiple-terraform-versions"
```
Comment (michalbagrowski) [inline test/test.tf]: will be fixed after https://github.com/aeternity/terraform-aws-aenode-deploy/pull/31 merge as master have issues

File: test/test.tf [hunk 3]
```diff
@@ -63,7 +63,7 @@ module "test_nodes_sydney_channels" {
 }
 
 module "test_lb_sydney" {
-  source                    = "github.com/aeternity/terraform-aws-api-loadbalancer?ref=master"
+  source                    = "github.com/aeternity/terraform-aws-api-loadbalancer?ref=test-on-multiple-terraform-versions"
```
Comment (michalbagrowski) [inline test/test.tf]: will be fixed after https://github.com/aeternity/terraform-aws-api-loadbalancer/pull/12/ merge as master have issues

**Review by dincho — CHANGES_REQUESTED** [review]
This should not into master as is.

**Review by dincho — APPROVED** [review]

**Review by venimus — APPROVED** [review]

**Pull request merged**